docs/targets/solana.rst

```diff
@@ -1,23 +1,52 @@
 Solana
 ======
 
-The Solana target requires `Solana <https://www.solana.com/>`_ v1.8.1.
-
-Solana has the following differences to Ethereum Solidity:
-
-- The address type is 32 bytes, not 20 bytes. This is what Solana calls an "account"
-- An address literal has to be specified using the ``address"36VtvSbE6jVGGQytYWSaDPG7uZphaxEjpJHUUpuUbq4D"`` syntax
-- There is no ``ecrecover()`` builtin function, but there is a ``signatureVerify()`` function which can check ed25519
-  signatures.
-- There is no concept of gas, so there is no gas functions
-- Solana balance is stored in a ``uint64``, so *address* ``.balance``, ``.transfer()`` and ``.send()``
-  all use ``uint64`` rather than ``uint256``.
-- Solana uses different accounts for the program code, and the contract data.
-- Programs are upgradable. There is no need to implement upgrades in Solidity.
+Solana Overview
+_______________
+
+The Solana blockchain has a unique account model that resembles a key-value database.
+As the underlying environment follows a programming model different than that of Ethereum, Solidity inner workings
+have been modified to work properly on Solana.
+
+A Solidity contract on Solana utilizes two accounts: a data account and a program account.
+The program account stores the contract's executable binary and owns the data account,
+which holds all the storage variables.
+
+Contract upgrades
++++++++++++++++++
+
+Provided that the data layout from a new contract is compatible with that of an old one, it is possible
+to update the binary in the program account and retain the same data, rendering
+contract upgrades implemented in Solidity unnecessary.
+
+Data types
+++++++++++
+
+- An account address consists of a 32-bits key, which is represented by the ``address`` type. This data model differs from
+  Ethereum 20-bits addresses.
+- Solana's virtual machine registers are 64-bit wide, so 64-bit integers ``uint64`` and ``int64`` are preferable over
+  ``uint256`` and `int256`. Likewise, the builtin functions for *address* ``.balance``, ``.transfer()`` and ``.send()``
+  use 64-bit integers.
+- An address literal has to be specified using the ``address"36VtvSbE6jVGGQytYWSaDPG7uZphaxEjpJHUUpuUbq4D"`` syntax.
+
+Runtime
++++++++
+
+- Each instruction on Solana has a fixed amount of compute units. This concept supersedes the idea of gas from
+  Ethereum, so gas related functions are not available on Solana.
```

Comment:
On ethereum, each instruction has a fixed number of gas units. This does not explain the difference between gas and compute units

Reply:
I have rephrased the sentence.

Reply:
It does not explain the fundamental difference between solana compute units and gas and why there are no gas or compute unit functions available. 

A developer with a solidity contract with gas functions is supposed to know how to re-write their contract without gas functions.